pkg/collector/corechecks/net/ntp/ntp.go

```diff
@@ -227,7 +228,7 @@ func (c *NTPCheck) queryOffset() (float64, error) {
 	}
 
 	if len(offsets) == 0 {
-		return .0, fmt.Errorf("Failed to get clock offset from any ntp host")
+		return .0, fmt.Errorf("Failed to get clock offset from any ntp host: %s", strings.Join(c.cfg.instance.Hosts, ", "))
```

Comment:
Is it possible `c.cfg.instance.Hosts` to be empty?

Reply:
No, it'll always have a default value `defaultHosts` which contains the `x.datadog.pool.ntp.org` hosts. 
But even if we change that, the function`strings.Join` produces an empty string on when given a nil slice.